gradle/publish.gradle

```diff
@@ -68,13 +68,11 @@ afterEvaluate {
     //from components.java
 
     // NOTE only build sources and javadoc jars when releasing
```

Comment:
Can you update this comment since you are now always generating sources & JavaDoc jars.